NAMESPACE

```diff
@@ -167,6 +197,7 @@ importFrom(dplyr,pull)
 importFrom(dplyr,rename)
 importFrom(dplyr,select)
 importFrom(dplyr,summarise)
+importFrom(dplyr,summarise_at)
```

Comment:
> Scoped verbs (_if, _at, _all) have been superseded by the use of across() in an existing verb. See vignette("colwise") for details.

Something to keep an eye on.

Reply:
Thanks for the heads up. I can't keep up with all these changes ;) I'll look into using across() instead.

Reply:
switched to using across and pushed so will mark this as resolved